src/main/java/mustapelto/deepmoblearning/common/tiles/TileEntityLootFabricator.java

```diff
@@ -194,7 +196,6 @@ public ByteBuf getUpdateData() {
     @Override
     public void handleUpdateData(ByteBuf buf) {
         super.handleUpdateData(buf);
-        pristineMatterMetadata = inputPristineMatter.getPristineMatterMetadata().orElse(null);
```

Comment:
It looks like pristineMatterMetadata is properly updated by ItemHandlerPristineMatter, and this line is unnecessary - but I'm not really sure.